### DISCUSSION TITLE
Migration (files only) Step 6 instructions unclear

### DISCUSSION THREAD

**Discussion by orrinwitt** [nextcloud/all-in-one#5819] [Questions] [answered]
### Is your feature request related to a problem? Please describe. 

For those of us who are still learning how to parse docker commands, the instructions in the migration.md document are a little hard to follow, specifically in step 6. Having mapped the NEXTCLOUD_DATADIR to /mnt/tank/nextcloud_data/ in the compose file, I have a hard time knowing how to adjust the chown/chmod commands in step 6. I have a working installation, but when I stop all the containers to run those commands in step 6 (which I'm probably doing wrong?), I get failures, and the nextcloud-aio-nextcloud container fails to fully start.

### Describe the solution you'd like

The other steps have clear instructions for exactly where and how to enter /path/to/nexcloud/data/ and I think this would be helpful for step 6 as well.

I realize that in step 5, the path being referenced is the location of the existing/old nextcloud data, and in step 6, the path being referred to is the new location for the AIO instance. Perhaps something like /path/to/old/nextcloud/data/ and /path/to/new/nextcloud/data/ to keep the distinction?

### Additional context

I can provide more info on specific errors, but I'm pretty sure this is the part I'm doing wrong.

**Comment by orrinwitt** [comment]
It seems like no matter how I enter the commands, I always end up with this error in the logs of the nextcloud-aio-nextcloud container:
```
2024-12-31T15:37:13.288748247Z -------------------------------
2024-12-31T15:37:13.288760998Z  2024-12-31 15:37:13.287772+00
2024-12-31T15:37:13.288771988Z (1 row)
2024-12-31T15:37:13.288793785Z 
2024-12-31T15:37:13.299164285Z + '[' -f /dev-dri-group-was-added ']'
2024-12-31T15:37:13.299619155Z ++ find /dev -maxdepth 1 -mindepth 1 -name dri
2024-12-31T15:37:13.300398454Z + '[' -n '' ']'
2024-12-31T15:37:13.300428592Z + set +x
2024-12-31T15:37:13.315648581Z Enabling Imagick...
2024-12-31T15:37:16.120979684Z WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.21/main: No such file or directory
2024-12-31T15:37:16.121035162Z WARNING: opening from cache https://dl-cdn.alpinelinux.org/alpine/v3.21/community: No such file or directory
2024-12-31T15:37:16.189768093Z Connection to nextcloud-aio-redis (172.16.4.5) 6379 port [tcp/redis] succeeded!
2024-12-31T15:37:16.571586945Z The initial Nextcloud installation failed.
2024-12-31T15:37:16.571634924Z Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
2024-12-31T15:37:16.571648235Z See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
2024-12-31T15:37:40.065490545Z Connection to nextcloud-aio-database (172.16.4.4) 5432 port [tcp/postgresql] succeeded!
2024-12-31T15:37:42.103594923Z               now              
2024-12-31T15:37:42.103658293Z -------------------------------
```

I don't mean for this to be asking for specific help in the wrong place, but there seems to be a real deficiency in the documentation on this point.

**Comment by szaimen** [comment]
> I realize that in step 5, the path being referenced is the location of the existing/old nextcloud data, and in step 6, the path being referred to is the new location for the AIO instance. Perhaps something like /path/to/old/nextcloud/data/ and /path/to/new/nextcloud/data/ to keep the distinction?

Hi, thanks for the idea! This was done in https://github.com/nextcloud/all-in-one/commit/99c5ae833633e068df30b1255de89372744f1571

**Comment by szaimen** [comment, marked as answer]
As for your other issue, you need to follow this:

> 2024-12-31T15:37:16.571586945Z The initial Nextcloud installation failed.
> 2024-12-31T15:37:16.571634924Z Please reset AIO properly and try again. For further clues what went wrong, check the logs above.
> 2024-12-31T15:37:16.571648235Z See https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance
